app/views/unattended/provisioning_templates/user_data/userdata_open_vm_tools.erb

```diff
@@ -9,8 +9,14 @@ oses:
 - Rocky
 - Debian
 - Ubuntu
+description: |
```

Comment:
@timogoebel could you please take a look and see if I covered that correctly? You also mention a finish template or cloud-init could be used after this, however is that possible today with Foreman? I think we can't chain multiple user-data templates for single OS nor we can combine finish and user-data. Also if you have any other comments for templates you know well, please add that. Thanks!

Reply:
Yes, we can use this mechanism and cloud-init combined. See https://github.com/theforeman/foreman_userdata (which is now part of core).
Note, that is is required as vsphere does not support running a finish script as part of the customization spec.

Reply:
See https://docs.theforeman.org/nightly/Provisioning_Guide/index-foreman-el.html#Provisioning_Virtual_Machines_in_VMware_vSphere-Provisioning_with_cloudinit_and_userdata_templates